docs/source/index.rst

```diff
@@ -0,0 +1,56 @@
+iohub
+~~~~~~~~~~~~~
+
+N-dimensional bioimaging produces data and metadata in various formats,
+and iohub aims to become a unified Python interface to the most common formats
+used at the Biohub and in the broader imaging community.
+
+
+Read
+====
+
+- OME-Zarr
+- Single-page TIFF, OME-TIFF, and NDTiff images written by Micro-Manager/pycro-manager
+- Custom data formats used by Biohub microscopes (e.g., PTI, Mantis (WIP), DaXi (TBD))
+
+Write
+======
+
+- OME-Zarr
+- Multi-page TIFF stacks organized in a directory hierarchy that mimics OME-NGFF (WIP)
+
+
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   :caption: Getting Started
+
+   Install <getting_started/install.rst>
+   Why iohub? <getting_started/why.rst>
+
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+   :caption: Tutorials
+
+   Tutorials Home <tutorials/home.rst>
```

Comment:
`index.rst:33: WARNING: toctree contains reference to nonexisting document 'tutorials/home`

Reply:
that's  section for future to curate tutorials but we can remove it if warnings are bothersome.

Reply:
Just wanted to confirm if its intentional. The warning is actually a reminder in this case.